src/services/chris_integration.tsx

```diff
@@ -147,17 +148,15 @@ class ChrisIntegration {
     let client: any = await ChrisAPIClient.getClient();
     try {
       console.log(img.fname)
+
+      // PL-DIRCOPY
       const dircopyPlugin = (await client.getPlugins({ "name_exact": this.FS_PLUGIN })).getItems()[0];
       // const params = await dircopyPlugin.getPluginParameters();
       const data: DirCreateData = { "dir": img.fname };
       const dircopyPluginInstance: PluginInstance = await client.createPluginInstance(dircopyPlugin.data.id, data);
+      console.log("PL-DIRCOPY task sent into the task queue")
 
-      const dirCopyResult = await pollingBackend(dircopyPluginInstance);
-      if (dirCopyResult.error) {
-        return [dirCopyResult];
-      }
```

Comment:
Since we are no longer waiting for each plugin to finish, it seems we also won't be checking for errors at each step. What happens when an intermediary plugin like `dircopy` or `med2img` fails? Does error message propagate to the last plugin for us to see on line 198? Or are we just going to receive a generic error message?

Reply:
Ok just to test this out, I tried removing the `med2img` plugin and running an analysis: it fails and returns the appropriate error message indicating that "pl-med2img not registered" in the notifications. This is definitely what we still want to see 👍

Reply:
Great questions @janakitti,

If in a chain one of the plugin fails, the subsequent plugins in the chain would also fail. Thus, by checking the result of the last plugin, we can identify whether the analysis has finished without errors or not.